nomenclature/definition.py

```diff
@@ -41,11 +44,18 @@ def __init__(self, path, dimensions=None):
         if not isinstance(path, Path):
             path = Path(path)
 
-        if (file := path.parent / "nomenclature.yaml").exists():
+        self.working_dir = path.parent

```

Comment:
I think the name of this variable is confusing. It's supposed to be the parent folder where the definition files are located, right? Normally, working directory is the directory where you call a script from.
Maybe project folder is a better name:
 ```suggestion
        self.project_folder = path.parent

```

Reply:
Nice...